Rakefile

```diff
@@ -4,3 +4,10 @@
 require_relative "config/application"
 
 Rails.application.load_tasks
+
+if defined?(Annotate)
```

Comment:
I had to add that to the Rakefile because, specifically with the Rails template I'm using, it won't automatically generate annotations with the Annotate gem. So, I included it as a sample task